react/components/HighlightOverlay/hooks/useStyles.ts

```diff
@@ -0,0 +1,125 @@
+import { CSSProperties, useMemo } from 'react'
+
+import { State } from '../typings'
+
+interface UseStylesArgs {
+  hasValidElement?: boolean
+  highlightTreePath: State['highlightTreePath']
+  visibleElement?: Element
+}
+
+let DEFAULT_HIGHLIGHT_RECT = { x: 0, y: 0, width: 0, height: 0 }
+
+const INITIAL_HIGHLIGHT_RECT = {
+  height: 0,
+  width: 0,
+  x: 0,
+  y: 0,
+}
+
+const BLUE = '#134CD8'
```

Comment:
:pick: Why not use one of our semantic tokens?

Reply:
Because I don't want the style of the store to change the the color of the higlight. 

In my first implementation, I used `action-primary` but peu asked me to change it to this blue color, and it makes sense since it's a VTEX application that's injecting the overlay.

This blue color is the same as the VTEX default `action-primary` but different from the one from the `blue` class, so I can't use `b--blue` for example.